### ISSUE TITLE
Sync/Connection speed plummets when web UI is open on PC

### ISSUE THREAD

**Issue by KarimElsayad247** [syncthing/syncthing#9512]
### What happened?

When Web GUI is running on my laptop, i.e, I'm in `http://127.0.0.1:8384/`, sync speed tops around a pathetic 1.12 MiB/s, but when close it, it jumps the the full expected high speed of over 15 MiB/s (over 300 MiB). 

Of course, no relay, not tunneling, no whatever, simple lovely TCP connection between 2 devices on the same LAN, both connected to the router via wifi.

Syncthing on Receiver is running in CLI (using `syncthing`

When Web GUI is closed on receiver browser, nothing is reported in console output, 

### Device and software details:

Device A (Sender): Android device 
- Xiaomi Mi 9T (MIUI Global 11.0.5 | Stable)
- Syncthing version 1.27.3
- Android 10 QKQ1.190825.002

Device B (Receiver): Linux Desktop/Laptop
- OS: Opensuse Tumbleweed 20240409
- Kernel version: 6.8.4-rc1-1-default (64-bit)
- Graphics Platform: X11
- KDE Plasma version: 6.0.3
- `syncthing --version`: ` v1.27.6 "Gold Grasshopper" (go1.21.9 linux-amd64) abuild@openSUSE 2024-04-09 09:23:58 UTC [noupgrade]`
- Browser: Microsoft Edge Version 123.0.2420.81 (Official build) (64-bit)

-----------------------------------

Issue was reported long ago 

https://github.com/syncthing/syncthing/issues/867

-------------------------------------

Please let me know if there is more useful information I could provide.
 

### Syncthing version

 v1.27.6 "Gold Grasshopper" (go1.21.9 linux-amd64) abuild@openSUSE 2024-04-09 09:23:58 UTC [noupgrade]

### Platform & operating system

linux-amd64, Opensuse Tumbleweed

### Browser version

Microsoft Edge Version 123.0.2420.81 (Official build) (64-bit)

**Comment by KarimElsayad247** [comment]
Hmmm Ok so this seems to have something related to wifi connection? Replacing the wifi connection with a connection over Tethered USB I get a friggin awesome 33.1 MiB/s (5.18 GiB) (sometimes over 40) with or without the web GUI opened.

**Comment by AudriusButkevicius** [comment]
This looks more like a support request than a clear bug or feature request. Please use the forum - https://forum.syncthing.net/ - for support, first by searching for any similar issues and then by posting a new one when required. The forum has a lot more eyes and friendly people on it than the issue tracker.

Thanks!

**Comment by KarimElsayad247** [comment]
@AudriusButkevicius In no way is this a support request. I have already solved my speed problems using USB tethering.

The purpose of this issue is to report some weird behavior I've found, and for which existed an issue 9 years ago with no clear resolution.

The previous thread already says something about it: having web ui uses resources so it's normal to see this slowdown, so the solution is to just close the web ui. Well, no when web ui is open, non of CPU cores are remotely close to 50%, leading me to believe there something wrong.
